chsdi/templates/htmlpopup/geothermische_potenzialstudien.mako

```diff
@@ -0,0 +1,84 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+<%
+    lang = lang if lang in ('fr','it','en') else 'de'
+    titel = 'titel_%s' % lang
+    weblink = 'weblink_%s' % lang
+    download = c['attributes']['download']
+%>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.land')}</td>       <td>${c['attributes']['land'] or '-'}</td></tr>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.kanton')}</td>       <td>${c['attributes']['kanton'] or '-'}</td></tr>
+     <tr><td class="cell-left">${_('ch.swisstopo.geologie-geothermische_potenzialstudien_regional.nb_studien')}</td>       <td>${c['attributes']['nb_studien'] or '-'}</td></tr>
```

Comment:
This returns "-" when the value in the table is "0". (see UR for example)
Should display "0" instead